src/broker/content-cache.c

```diff
@@ -645,7 +645,7 @@ void content_checkpoint_get_request (flux_t *h, flux_msg_handler_t *mh,
     struct content_cache *cache = arg;
     const char *topic = "content-backing.checkpoint-get";
     const char *s = NULL;
-    const flux_msg_t *msgcpy = flux_msg_incref (msg);
+    const flux_msg_t *msgref = flux_msg_incref (msg);
```

Comment:
I suggest instead of renaming the variable, we get rid of it and fix the refcounting problem on the error paths, as discussed in the other PR.

While you're at it, may as well fix the future leak that occurs when aux_set or then fails.